xarray/tests/test_duck_array_ops.py

```diff
@@ -1094,3 +1095,19 @@ def test_extension_array_singleton_equality(categorical1):
 def test_extension_array_repr(int1):
     int_duck_array = PandasExtensionArray(int1)
     assert repr(int1) in repr(int_duck_array)
+
+
+def test_extension_array_attr():
+    array = pd.Categorical(["cat2", "cat1", "cat2", "cat3", "cat1"])
+    wrapped = PandasExtensionArray(array)
+    assert_array_equal(array.categories, wrapped.categories)
+    assert array.nbytes == wrapped.nbytes
+
+    roundtripped = pickle.loads(pickle.dumps(wrapped))
+    assert isinstance(roundtripped, PandasExtensionArray)
+    assert (roundtripped == wrapped).all()
```

Comment:
What is this testing?

Reply:
that we can roundtrip with pickle.

Reply:
Was this broken previously? Don't know much about pickling, just asking because of the name of the test, i.e., testing for different attrs

Reply:
pickling uses `__getstate__` and `__setstate__` so I';m making sure that isn't broken

Reply:
got it, makes sense!